Hi3Helper.Core/Lang/en.json

```diff
@@ -438,79 +438,79 @@
 
   "_Dialogs": {
     "DeltaPatchDetectedTitle": "Warning: Delta Patch Update is Detected!",
-    "DeltaPatchDetectedSubtitle": "Delta patch will be used to update your game from {0} to {1}.\r\nRemember, you might still need to download some files in-game as usual.\r\n\r\nDo you want to proceed?",
-    "DeltaPatchPrevFailedTitle": "Previous Delta Patch was Failed/Incompleted!",
-    "DeltaPatchPrevFailedSubtitle": "The previous Delta Patch failed to apply. Do you want to restore your game to prevent from redownloading?",
-    "GameConversionPrevFailedTitle": "Previous Game Conversion was Failed/Incompleted!",
+    "DeltaPatchDetectedSubtitle": "Delta patch will be used to update your game from {0} to {1}.\r\nRemember that you may still need to download some files in-game.\r\n\r\nDo you wish to proceed?",
+    "DeltaPatchPrevFailedTitle": "Previous Delta Patch has been incorrectly applied or failed to finish applying!",
+    "DeltaPatchPrevFailedSubtitle": "The previous Delta Patch failed to apply. Do you want to restore your game to prevent a redownload?",
+    "GameConversionPrevFailedTitle": "Previous Game Conversion has Failed or was Incompleted!",
     "GameConversionPrevFailedSubtitle": "The previous Game Conversion failed. Do you want to restore your game to prevent from redownloading?",
     "PreloadVerifiedTitle": "Pre-load Package Verified!",
     "PreloadVerifiedSubtitle": "Your Pre-load Package is Ready and was Verified!",
     "LocateInstallTitle": "Locating Installation Folder",
     "LocateInstallSubtitle": "Before Installing the Game, Do you want to specify the location of the game?",
     "UnauthorizedDirTitle": "Unauthorized Location Chosen",
-    "UnauthorizedDirSubtitle": "You have chosen a location which you don't have permission to write to at this path:\n\n{0}\n\nPlease choose another location!",
+    "UnauthorizedDirSubtitle": "You have chosen a location in which you do not have permission to write:\n\n{0}\n\nPlease choose another location!",
```

Comment:
Simpler sentence by making it like
`You don't have write permission on this location:\n\n{0}\n\nPlease choose another location!`

Reply:
I'm agree with this. Should be more simplified

Reply:
Rephrasing it `You don't have permission to write in:\n\n{0}\n\nPlease choose another location!` to make it grammatically correct.